pkg/arvo/app/chat-cli.hoon

```diff
@@ -741,15 +741,22 @@
     ::  +create: new local mailbox
     ::
     ++  create
-      |=  [security=nu-security =path gyf=(unit char) allow-history=(unit ?)]
+      |=  $:  security=nu-security
+              =path
+              ugroup=(unit resource)
+              gyf=(unit char)
+              allow-history=(unit ?)
+          ==
       ^-  (quip card _state)
-      =/  with-group=?     ?=(%village-with-group security)
+      =/  with-group=?     ?=(^ ugroup)
       =/  =target          [with-group our-self path]
       =/  real-path=^path  (target-to-path target)
+      =/  group-path=^path  ?~(ugroup ship+real-path (en-path:resource u.ugroup))
+      ?<  &(?=(%channel security) with-group)
```

Comment:
The parser still allows this, right? You might want to put a `+sear` or whatever around it to ensure people cannot trigger this assertion. That's not gonna make for the prettiest parser though...